src/utils/pbconvert.hpp

```diff
@@ -42,21 +42,34 @@ constexpr void PBToEnum(const char (&src)[cSize], T& dst)
  * Converts Aos error to PB error info.
  *
  * @param aosError Aos error to convert from.
- * @param[out] errorInfo PB error info to convert to.
+ * @return common_v1_ErrorInfo.
  */
-template <typename T>
-void ErrorToPB(const Error& aosError, T& errorInfo)
+inline common_v1_ErrorInfo ErrorToPB(const Error& aosError)
 {
-    if (aosError.IsNone()) {
-        errorInfo.has_error = false;
-        return;
+    common_v1_ErrorInfo errorInfo = {static_cast<int32_t>(aosError.Value()), aosError.Errno()};
+
+    StringFromCStr(errorInfo.message).Convert(aosError);
+
+    return errorInfo;
+}
+
+/**
+ * Converts Aos error to PB error info.
+ *
+ * @param errorInfo PB error info to convert from.
+ * return Error.
+ */
+inline Error PBToError(const common_v1_ErrorInfo& errorInfo)
```

Comment:
Should you convert PB error info message as well?

Reply:
Added